README.md

```diff
@@ -0,0 +1,74 @@
+# Amazon S3 Persistance for OpenXava using Java Reflection
+
+## Introduction:
+  - Openxava has two ways of storing a file. 
+    - File system persistor which stores the file in the server/local. 
+    - JPAFilePersistor which stores file in the database. 
+
+  - The most common practice used these days to persist document/files is to store it in amazon S3 storage which provides very high reliability, durability, and security. 
+
+  - In addition to the current two methods this patch introduces ```Amazon S3 persistence```
+  
+  - We have used Java Reflection API that is used to examine or modify the behaviour of methods, classes and interfaces at runtime. 
+  - The required packages for Reflection, the “java.lang.reflect” package, allows us to invoke methods at runtime irrespective of the access modifier.
+  
+  - From Java reflection we removed the dependency of having the required jars during compile time. 
+  
+  - Now since the dependency of jars not required. Additonal jars are required only if you enable ```amazonS3Persistor``` in xava.properties
```

Comment:
The readme file should be the text which would go into a OpenXava documentation section. The details like reflection method is too technical to be included here. Pull request description may mention these content as implementation details.